js/utils/metrics.js

```diff
@@ -0,0 +1,111 @@
+import MetricsModal from '../views/modals/MetricsModal';
+import app from '../app';
+import { version } from '../../package.json';
+import * as os from 'os';
+import { getCurrentConnection } from './serverConnect';
+import { remote } from 'electron';
+
+
+let metricsRestartNeeded = false;
+
+export function isMetricRestartNeeded() {
+  return metricsRestartNeeded;
+}
+
+export function addMetrics() {
+  function loadMetrics() {
+    // Reverse the countly opt out
+    window.localStorage.setItem('cly_ignore', 'false');
+    metricsRestartNeeded = !!window.Countly;
+
+    //  If Countly has already been added, it won't run again until the app is restarted.
+    if (window.Countly) return;
+
+    const sVer = app.settings.get('version');
+    const serverVersion = sVer.substring(sVer.lastIndexOf(':') + 1, sVer.lastIndexOf('/'));
```

Comment:
Looks like this is also duplicated in the about modal. Can we instead add a method to the settings model to compute this...maybe a `prettyServerVer()` getter?